app/models/patron.rb

```diff
@@ -37,8 +37,8 @@ def display_name
 
   def checkouts
     @checkouts ||= fields['circRecordList']
-      .map { |checkout| Checkout.new(checkout) }
-      .select { |checkout| checkout.status == 'ACTIVE' }
+                     &.map { |checkout| Checkout.new(checkout) }
+                     &.select { |checkout| checkout.status == 'ACTIVE' }
```

Comment:
Curious why you added safe navigation here? Does rails throw if there are no checkouts?

Reply:
One of the specs I added were failing because checkouts were empty for that case I was testing.

Reply:
Love the safe navigation operator, so convenient